corehq/apps/hqadmin/templates/hqadmin/admin_restore.html

```diff
@@ -83,7 +86,7 @@ <h3>Response Status Code: {{ status_code }}</h3>
                         <div class="tab-content">
                             <div role="tabpanel" class="tab-pane active" id="payload-tab">
                                 {% if not hide_xml %}
-                                    <pre id="payload">{{ payload }}</pre>
+                                    <pre id="payload" data-payload="{{ payload }}"></pre>
```

Comment:
Should this be `{% html_attr payload %}` to take care of potential special characters?

Reply:
probably. updating.